### DISCUSSION TITLE
RAE2822 airfoil

### DISCUSSION THREAD

**Discussion by AminNRC** [mdolab/dafoam#447] [Tutorials]
Hello, 
The current folder of the "RAE2822 airfoil" in the "validations" section only generates the medium mesh with 60,672 cells. How can I generate the exact coarse and fine meshes used in the DAFoam validation analysis having 17,408 and 108,675 cells, respectively? Thank you very much for your help.

**Comment by friedenhe** [comment]
Please check the DAFoam website FAQ on how to refine the mesh for this case.
